Frequently_used_code/ARD_overpass_predictor.ipynb

```diff
@@ -0,0 +1,1677 @@
+{
```

Comment:
<p>I think you can assume that the user will change their input file if they want to analyse somewhere else. Instead of telling the user to change the file, explain how the file is formatted (which you've done below anyway) and they can make that change if they want. So I reckon remove the "Caution" line.</p><p> </p><p>The input file is <code>csv</code> now right, not <code>xlsx</code> ?</p><p> </p><p>I don't quite understand this line, as I think I should be able to run the notebook without making nchanges:</p><blockquote>Make changes to the notebook, following the <strong>Steps</strong> in bold</blockquote><p> </p><p>Load packages shouldn't be a subsection of description, it should be in a getting started section - check the template and make sure you're matching how it is formatted and structured.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/736/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>